src/poliastro/earth/sensors.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+from astropy import units as u
+
+from poliastro.bodies import Earth
+
+
+@u.quantity_input(h=u.km, n_fov=u.rad, n_center=u.rad)
+def max_and_min_ground_range(h, n_fov, n_center):
+    """
+    Calculates the minimum and maximum values of ground-range angles
+
+    Parameters
+    ----------
+    h: ~astropy.units.Quantity
+        Altitude over surface.
+    n_fov: ~astropy.units.Quantity
+        Angle of the total area that a sensor can observe.
+    n_center: ~astropy.units.Quantity
+        Center boresight angle.
+
+    Returns
+    -------
+    Λ_min: ~astropy.units.Quantity
+        Minimum value of latitude and longitude.
+    Λ_max: ~astropy.units.Quantity
+        Maximum value of latitude and longitude.
+
+    Notes
+    -----
+    For further information, please take a look at "Fundamentals of Astrodynamics and Applications", 4th ed (2013)"
+    by David A. vallado, pages 853-860.
+
+    """
+
+    R = Earth.R.to(u.km)
+    r_sat = R + h
+    n_max = n_center + n_fov / 2
+    n_min = n_center - n_fov / 2
+    γ_max = np.arcsin(r_sat * np.sin(n_max) / R)
+
+    if abs(γ_max) <= np.pi / 2 * u.rad:
+        γ_max = np.pi * u.rad - γ_max
+    γ_min = np.arcsin(r_sat * np.sin(n_min) / R)
+    if abs(γ_min) <= np.pi / 2 * u.rad:
+        γ_min = np.pi * u.rad - γ_min
+
+    p_max = R * np.cos(γ_max) + r_sat * np.cos(n_max)
```

Comment:
Let's add a comment here saying "maximum and minimum slant ranges" (since `ρ` is not defined in the inputs nor the outputs)